cannot build exif on osx

## Issue description

nix-env -i exif  failed on osx 10.9.5

### Steps to reproduce

```
$ nix-env -i exif
...
/nix/store/pqkmz8xbf1xg40dbr79rkcmcqcmx8hhf-bash-4.3-p42/bin/bash ../libtool  --tag=CC   --mode=link clang  -g -O2 -g -Wall  -o exif exif-actions.o exif-exif-i18n.o exif-main.o exif-utils.o -g -Wall  ../libjpeg/libjpeg.la -L/nix/store/ydqvdhkzkixgdaqm8psp6n09s6wqj4i5-libexif-0.6.21/lib -lexif -L/usr/ -lpopt  
libtool: link: clang -g -O2 -g -Wall -o exif exif-actions.o exif-exif-i18n.o exif-main.o exif-utils.o -g -Wall  ../libjpeg/.libs/libjpeg.a -L/nix/store/ydqvdhkzkixgdaqm8psp6n09s6wqj4i5-libexif-0.6.21/lib /nix/store/ydqvdhkzkixgdaqm8psp6n09s6wqj4i5-libexif-0.6.21/lib/libexif.dylib -lintl -lm -L/usr/ -lpopt
ld: library not found for -lintl
clang-3.7: error: linker command failed with exit code 1 (use -v to see invocation)
Makefile:342: recipe for target 'exif' failed
make[2]: *** [exif] Error 1
make[2]: Leaving directory '/private/var/folders/cy/4988f14n4r35fp39wrt1pfq80000gn/T/nix-build-exif-0.6.21.drv-0/exif-0.6.21/exif'
Makefile:409: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/private/var/folders/cy/4988f14n4r35fp39wrt1pfq80000gn/T/nix-build-exif-0.6.21.drv-0/exif-0.6.21'
Makefile:297: recipe for target 'all' failed
make: *** [all] Error 2
builder for ‘/nix/store/2wmwfh1b6scnfc73a0h25g186d27fnla-exif-0.6.21.drv’ failed with exit code 2
error: build of ‘/nix/store/2wmwfh1b6scnfc73a0h25g186d27fnla-exif-0.6.21.drv’ failed
$
```

## Technical details

System: Darwin Kernel Version 13.4.0: Mon Jan 11 18:17:34 PST 2016; root:xnu-2422.115.15~1/RELEASE_X86_64 x86_64
Nix version: nix-env (Nix) 1.11.2
Nixpkgs version: "16.09pre83147.df89584"

* System: (NixOS: `nixos-version`, Ubuntu/Fedora: `lsb_release -a`, ...): Darwin Kernel Version 13.4.0: Mon Jan 11 18:17:34 PST 2016; root:xnu-2422.115.15~1/RELEASE_X86_64 x86_64
* Nix version: (run `nix-env --version`): nix-env (Nix) 1.11.2
* Nixpkgs version: (run `nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion`): "16.09pre83147.df89584"

--- a/pkgs/tools/graphics/exif/default.nix
+++ b/pkgs/tools/graphics/exif/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, pkgconfig, libexif, popt }:
+{ stdenv, fetchurl, pkgconfig, libexif, popt, libintlOrEmpty }:
 
 stdenv.mkDerivation rec {
   name = "exif-0.6.21";
@@ -8,8 +8,8 @@ stdenv.mkDerivation rec {
     sha256 = "1zb9hwdl783d4vd2s2rw642hg8hd6n0mfp6lrbiqmp9jmhlq5rsr";
   };
 
-  buildInputs = [ pkgconfig libexif popt ];
-  
+  buildInputs = [ pkgconfig libexif popt ] ++ libintlOrEmpty;
+
   meta = {
     homepage = http://libexif.sourceforge.net/;
     description = "A utility to read and manipulate EXIF data in digital photographs";

